examples/ffi/trace_exporter.c

```diff
@@ -57,7 +50,9 @@ int main(int argc, char** argv)
     }
 
     ddog_ByteSlice buffer = { .ptr = NULL, .len=0 };
-    TRY(ddog_trace_exporter_send(trace_exporter, buffer, 0));
+    ddog_AgentResponse response;
+
+    TRY(ddog_trace_exporter_send(trace_exporter, buffer, 0, &response));
```

Comment:
/suggestion

since we are not sending the error information in return, we should update the example to check the error and freeing the error if returned.